src/test/java/org/ambraproject/wombat/service/PeerReviewServiceTest.java

```diff
@@ -92,6 +99,11 @@ String getReviewXml(Map<String, ?> metadata) throws IOException {
               "<front-stub><custom-meta-group><custom-meta><meta-name>Submission Version</meta-name><meta-value>1</meta-value></custom-meta></custom-meta-group></front-stub>" +
               "<body><p>FirstRoundDecisionLetterSampleBody</p></body></sub-article>";
         }
+        if (crepoKey == "info:doi/10.1371/journal.pone.0207232.r004.xml") {
+          letterContent = "<?xml version=\"1.0\" encoding=\"utf-8\"?><sub-article specific-use=\"acceptance-letter\">" +
+              "<front-stub></front-stub>" +
+              "<body><p>AcceptanceLetterSampleBody</p></body></sub-article>";
```

Comment:
I just noticed these extra `<p>` and `</p>` tags in the mark-up, which is not a huge deal, except for when fonts are relatively sized.  Take a look at the screenshot for the tags I'm referring to.  Any way we can remove them?

<img width="1122" alt="ambr-833" src="https://user-images.githubusercontent.com/25396501/52295965-03c73000-2932-11e9-9406-7ac4bc70d6e4.png">

Reply:
Should be straightforward, thank you!

Reply:
It was straightforward.